UnityProject/Assets/Scripts/Messages/Server/ShootMessage.cs

```diff
@@ -49,7 +48,7 @@ public override void Process()
 			//Not even spawned don't show bullets
 			if (PlayerManager.LocalPlayer == null) return;
 
-			LoadMultipleObjects(new [] { Shooter, Projectile });
+			LoadMultipleObjects(new uint[] { Shooter, Weapon });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6395121)